### ISSUE TITLE
"Sequence" is an option for imports, no longer a Person attribute

### ISSUE THREAD

**Issue by zarniwoop** [churchio/onebody#672]
While importing from a CSV file, I wanted to provide the ordinal information for each family member so they would show up with parents first and then siblings from oldest to youngest.  I initially selected "sequence" from the list of fields I could import into, but when I continued with the import I got an error saying sequence was not an attribute of Person.  After some searching of the code I realized that sequence had been renamed to "position" at some point.  I then selected "position" and everything worked.

Bottom line: "sequence" should be removed from the list of possible fields to import into.

**Comment by seven1m** [comment]
Good catch!
